.gitignore

```diff
@@ -53,3 +53,6 @@ fastlane/Preview.html
 fastlane/screenshots
 
 TODO.md
+
+realm-object-server/
+default.realm*
```

Comment:
A comment to say what these files are and how they will be generated would be handy :)